test-setup/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "test-setup"
+version = "0.1.0"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+
+[dependencies]
+bytes = "1.5.0"
+imap-codec = "2.0.0"
+imap-flow = { path = ".." }
+imap-types = "2.0.0"
+tokio = { version = "1.32.0", features = ["full"] }
```

Comment:
suggestion: We could use a more minimal feature set to spare us a bit compile time.

Reply:
Yes